src/test/java/com/scottlogic/AuthorPostSorterTest.java

```diff
@@ -0,0 +1,93 @@
+package com.scottlogic;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.Arrays;
+import java.util.List;
+
+public class AuthorPostSorterTest {
+
+    AuthorPostSorter sortByAuthor = new AuthorPostSorter();
+    private UserPost createUserPost (String name ) {
+
+        return new UserPost(name,
+                OffsetDateTime.of(2020, 1, 3, 7, 12, 3, 0, ZoneOffset.UTC),
+                "Hello World!", 2);
+    }
+
+
+
+
+
+    @Test
+    public void authorSorter_mixedValidNames_correctListReturnedAsc() {
+        UserPost userPost1 = createUserPost("Joe Bloggs");
+        UserPost userPost2 = createUserPost("Joe Bloggs");
```

Comment:
Nice job including two posts with the same author. This could probably be a test in its own right though - I'm generally keen on testing "minimal" scenarios.